store/hbase/txn.go

```diff
@@ -103,6 +103,18 @@ func (txn *hbaseTxn) Get(k kv.Key) ([]byte, error) {
 	return val, nil
 }
 
+func (txn *hbaseTxn) BatchGet(keys []kv.Key) (map[string][]byte, error) {
+	encodedKeys := make([]kv.Key, len(keys))
+	for i, k := range keys {
+		encodedKeys[i] = kv.EncodeKey(k)
+	}
+	return txn.UnionStore.Snapshot.BatchGet(encodedKeys)
+}
+
```

Comment:
Yes, we may get wrong values here. As a caller of Transaction actually is not really care about specific values, I will change function names of Transaction from `BatchGet` to `BatchPrefetch`.